src/Resources/Locales/en.json

```diff
@@ -652,6 +652,7 @@
                     "labelPlaceholder": "Enter label to show in chart",
                     "add": "Add time series",
                     "edit": "Edit time series",
+                    "nonNumericWarning": "Note: Non-numeric properties will be cast to numeric at query time, and may result in some values being omitted",
```

Comment:
something is not right here. Why don't we see all the language files getting auto generated?